ui/scss/component/_button.scss

```diff
@@ -156,6 +156,10 @@ svg + .button__label,
   }
 }
 
+.button-toggle--more {
+  margin-bottom: var(--spacing-s);
```

Comment:
This causes unequal spacing below the buttons in the header. They should have the same top/bottom margin. Maybe the search bar should go full-width on small screens? Then you could add a margin-top to it.

Reply:
I've updated the PR with a different approach, where each element will have vertical spacing for better reflow.  See if that works.

Reply:
Nice. Looks great!